core/src/modules/ops/export.ts

```diff
@@ -9,6 +9,8 @@ import { CLS } from "../../modules/cls";
 import { Op } from "sequelize";
 
 export namespace ExportOps {
+  const defaultExportProcessingDelay = 1000 * 60 * 5;
```

Comment:
What's the behavior of this approach if an export is enqueued and it doesn't get processed within 5 minutes? Maybe there aren't enough workers or another long run comes up or the workers are stoped for devops reasons.

So the first one is still in the queue. Will this put another one in the queue or is there a queue lock in place and so this just, in effect, bumps the start time?

Reply:
We've got QueueLocks!  

It gets weird though with export batches - if the batch collection is different, the lock won't prevent the export from being included twice.  

I think this means the default delay should be higher - I'll bump that in the settings to be 6 hours

Reply:
We can fix that on the other side too then. In the beginning of the export jobs, do we check that they haven't been successfully completed? Then, if they are in the twice, the second one is a no-op. That narrows the window down to just the runtime.

Reply:
Nice idea!

Reply:
oh hey! We already do - https://github.com/grouparoo/grouparoo/blob/main/core/src/tasks/export/send.ts#L22-L25 and https://github.com/grouparoo/grouparoo/blob/main/core/src/tasks/export/sendBatch.ts#L25-L31

Reply:
With ^ in mind, I went back down to a 30 min delay

Reply:
Nice. if this still poses an issue, we'd en up full delayed_job and get a `processingAt` column